ui/src/components/table/tests/CustomTableRow.test.tsx

```diff
@@ -0,0 +1,112 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+
+import { BrowserRouter } from 'react-router-dom';
+import { http, HttpResponse } from 'msw';
+import { TableContextProvider } from '../../../context/TableContext';
+import { server } from '../../../mocks/server';
+import { setUnifiedConfig } from '../../../util/util';
+import { SIMPLE_NAME_TABLE_MOCK_DATA_WITH_STATUS_TOGGLE_CONFIRMATION } from '../stories/configMockups';
+import {
+    MockRowData,
+    MockRowDataTogglingResponseDisableFalse,
+    MockRowDataTogglingResponseDisableTrue,
+} from '../stories/rowDataMockup';
+import TableWrapper, { ITableWrapperProps } from '../TableWrapper';
+
+const handleRequestModalOpen = jest.fn();
+const handleOpenPageStyleDialog = jest.fn();
+
+beforeEach(() => {
+    const props = {
+        page: 'inputs',
+        serviceName: 'example_input_one',
+        handleRequestModalOpen,
+        handleOpenPageStyleDialog,
+        displayActionBtnAllRows: false,
+    } satisfies ITableWrapperProps;
+
+    server.use(
+        http.get('/servicesNS/nobody/-/splunk_ta_uccexample_example_input_one', () =>
+            HttpResponse.json(MockRowData)
+        ),
+        http.post('/servicesNS/nobody/-/splunk_ta_uccexample_example_input_one/aaaaaa', () =>
+            HttpResponse.json(MockRowDataTogglingResponseDisableTrue)
+        )
+    );
+
+    setUnifiedConfig(SIMPLE_NAME_TABLE_MOCK_DATA_WITH_STATUS_TOGGLE_CONFIRMATION);
+
+    render(
+        <TableContextProvider>
+            <TableWrapper {...props} />
+        </TableContextProvider>,
+        { wrapper: BrowserRouter }
+    );
+});
+
+it('Status toggling with acceptance model displayed correctly', async () => {
+    const inputSwitches = await screen.findAllByRole('switch');
+
+    // open accept modal for first switch
+    await userEvent.click(inputSwitches[0]);
+
+    const acceptModal = await screen.findByRole('dialog');
```

Comment:
You can search by role dialog with a name `Make input Inactive?`

<img width="1411" alt="image" src="https://github.com/user-attachments/assets/9264c3ba-8f6c-4176-9473-19d546a77df4">